build/kubernetes/coredns-test.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: coredns-test
+  namespace: kube-system
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: coredns-test
+  namespace: kube-system
+data:
+  Corefile: |
+    .:53 {
+        errors
+        forward . 10.96.0.10:53
+    }
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: coredns-test
+  namespace: kube-system
+  labels:
+    k8s-app: coredns-test
+    kubernetes.io/name: "CoreDNS-test"
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      k8s-app: coredns-test
+  template:
+    metadata:
+      labels:
+        k8s-app: coredns-test
+    spec:
+      serviceAccountName: coredns-test
+      tolerations:
+      - key: "CriticalAddonsOnly"
```

Comment:
Should go through this deployment spec and clean up stuff that isn't needed...
some cruft here can be removed.  e.g. e.g. tolerations ...  resources ... node selector ... update strategy... service account...

Reply:
Yes, this manifest still needs fine-tuning. I want to make a separate PR to modify this to have the ability to spin up many `coredns-test pods` when required.